src/scripts.js

```diff
@@ -126,4 +128,54 @@ function updateActivityDayHTML(date) {
     <h2>Global Average Today:</h2>
     <p>Steps: ${glabalAverageNumSteps}, Minutes Active: ${glabalAverageMinutesActive}, flights Of Stairs: ${glabalAverageFlightsOfStairs}</p>`
   );
+}
+
+function updateActivityWeekHTML(date) {
+  let activityForWeek = currentActivityUser.getActivityInfoForPreviousSevenDays(date);
+  activityForWeek.map(day => {
+    activityWeekly.insertAdjacentHTML('beforeend', `<p>${day.date}: - minutesActive: ${day.minutesActive}, flights of stairs: ${day.flightsOfStairs}</p>`)
+  });
+  generateChart(weeklyActivityGraph, activityForWeek, "numSteps");
+}
+
+function generateChart(node, weekOfData, property) {
```

Comment:
This function allows for charts to be displayed in a lot fewer lines thanks to arguments! Looks great, pulling now. Good work on this project!!